cloudify_nsx/library/nsx_common.py

```diff
@@ -181,6 +180,25 @@ def remove_properties(name):
         del ctx.instance.runtime_properties[name]
 
 
+def attempt_with_rerun(func, **kwargs):
+    """Rerun func several times, useful after dlr/esg delete"""
+    i = 10
```

Comment:
We are not ignoring, only recheck ten times before raise Recoverable Exception. 

Reason of it - any esg and dlr is vm, and can return success before really drop vm, so lswich can't be deleted before full drop esg/dlr vm.